jstz_api/src/urlpattern/mod.rs

```diff
@@ -0,0 +1,576 @@
+//! `jstz`'s implementation of JavaScript's `URLPattern` Web API.
+//!
+//! More information:
+//!  - [MDN documentation][mdn]
+//!  - [WHATWG `URLPattern` specification][spec]
+//!
+//! [mdn]: https://developer.mozilla.org/en-US/docs/Web/API/URLPattern
+//! [spec]: https://urlpattern.spec.whatwg.org/
+
+use boa_engine::{
+    js_string,
+    object::{builtins::JsArray, Object},
+    property::Attribute,
+    value::TryFromJs,
+    Context, JsArgs, JsError, JsNativeError, JsObject, JsResult, JsString, JsValue,
+    NativeFunction,
+};
+use boa_gc::{custom_trace, Finalize, GcRefMut, Trace};
+use jstz_core::{
+    accessor,
+    native::{
+        register_global_class, Accessor, ClassBuilder, JsNativeObject,
+        JsNativeObjectToString, NativeClass,
+    },
+    value::IntoJs,
+};
+use urlpattern::quirks::StringOrInit as InnerStringOrQuirksInit;
+use urlpattern::quirks::UrlPatternInit as InnerUrlPatternQuirksInit;
+use urlpattern::UrlPattern as InnerUrlPattern;
+use urlpattern::UrlPatternComponentResult as InnerUrlPatternComponentResult;
+use urlpattern::UrlPatternResult as InnerUrlPatternResult;
```

Comment:
Nit: All `use`s from a single crate should be a single line

Reply:
Done.